### PR TITLE
STR-60 -- FE - Create alcohol module thumbnail

### PR DESCRIPTION
**Pull request by frlzjosh**
# Description
'Write a meaningful description of your task'
  Create the alcohol module thumbnail
# Testing
'What does the reviewer need to do to test your PR?'
 1. Make sure to login as someone with module thumbail. I've been using `april.feldman`
 2. Look at my code make sure it's good, & give help/critique
 3. Make sure it is mobile responsive and pleasing to the eye
 4. Make sure it is close to the mockup. 
# Installation
'Are there any dependencies the reviewer needs to download?'
If you do not see the progress bar loading, then you will need to run `yarn install`

### CODE REVIEW

**Review by mbendandour — COMMENTED** [review]
The photos don't work on my MAMP machine, but the days and completion shows up and looks nice

File: resources/js/components/dashboard/DashboardModule.vue [hunk 1]
```diff
@@ -4,17 +4,31 @@
...
             <div class="dashboard-module__header">
               {{item.name}}
             </div>
           </div>
-          <div class="dashboard-module__progress row align-items-center">
+          <div class="dashboard-module__body row align-items-center">
+            <div class="col-2 text-left">
+              <div class="dashboard-module__body-progress">
+                <p class="dashboard-module__body-text"
+                 :class="item.completion.progress.toString().length <= 2 ?
```
Comment (andrewMETALAB) [inline resources/js/components/dashboard/DashboardModule.vue]: cool inline ternary statement!  and perfect usage of BEM modifier!

**Review by andrewMETALAB — CHANGES_REQUESTED** [review]
Overall this is great work. 

You broke BEM convention just a little, but it should be pretty easy to fix...
`dashboard-module__body` is an element (which is legit), but then you have other _"elements of that element"_ such as `dashboard-module__body-progress` and `dashboard-module__body-text`. You aren't supposed to have _"elements of elements"_ like that. So instead of naming them like that, they should all just be elements of `dashboard-module` So basically update them to  `dashboard-module__progress` and `dashboard-module__text` and so on. 

Ok now for your padding and margin helper classes! I actually think this is pretty cool and could be helpful. But I think we should use `rems` instead of `px` values. Also do you think we are "stepping on the toes" of bootstrap by creating these, since bootstrap offers something similar? Maybe we should keep the naming consistent with bootrap's classes? Read this and let me know what you think: https://getbootstrap.com/docs/4.3/utilities/spacing/

**Review by andrewMETALAB — APPROVED** [review]
Very well done 👍 👍 👍

**Review by BrianLinggadjaja — APPROVED** [review]
Awesome!

**Pull request merged**